app/export/courses.rb

```diff
@@ -0,0 +1,32 @@
+# Takes an existing semester live on umd.io and exports it to JSON
+# TODO: Update to use v1 schema, but only when courses_importer does too
+# TODO: Use logger
+# TODO: Config to target localhost:3000 instead - decide on convention for all exporters to follow
+
+require 'open-uri'
+require 'json'
+
+def export_sem(sem)
+  courses = []
+  n = 0
+
+  while (res = open("https://api.umd.io/v0/courses?semester=#{sem}&expand=sections&page=#{n}").read)
```

Comment:
wouldn't it be easier to pull data straight from the database?

Reply:
Sure, but I think there's also value in having a script like this to pull a copy of a running instance. I plan to improve this in future PRs